aiter/ops/triton/mha.py

```diff
@@ -285,7 +289,7 @@ def _attn_fwd_inner(
         if IS_CAUSAL:
             causal_boundary = start_n + offs_n_causal
             causal_mask = OFFS_M[:, None] >= causal_boundary[None, :]
-            mask = mask and causal_mask
+            mask = mask & causal_mask
```

Comment:
Just out of curiosity, does it help with the perf?

Reply:
I think the "and" generates deprecation warning msg .